theia-extensions/theia-blueprint-updater/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "private": true,
   "name": "theia-blueprint-updater",
-  "version": "1.19.0",
+  "version": "1.20.0",
   "description": "Eclipse Theia Blueprint Updater",
   "dependencies": {
-    "@theia/core": "1.19.0",
-    "@theia/output": "1.19.0",
-    "@theia/preferences": "1.19.0",
+    "@theia/core": "1.20.0",
+    "@theia/output": "1.20.0",
+    "@theia/preferences": "1.20.0",
     "electron-log": "^4.3.0",
-    "electron-updater": "^4.3.5",
+    "electron-updater": "4.3.3",
```

Comment:
Is this downgrade a result of running `yarn audit`?

Reply:
No, it's because more recent versions, getting pulled during the upgrade, were causing (IIRC build) issues.

Reply:
I quickly reproduced to refresh my mind. It was a startup issue actually, at least for the .AppImage package: 
```
theia-blueprint$ ./applications/electron/dist/TheiaBlueprint.AppImage 
Failed to start the electron application.
Error: Cannot find module 'fs/promises'
Require stack:
- /tmp/.mount_TheiaBXVxquD/resources/app/node_modules/electron-updater/out/AppUpdater.js
- /tmp/.mount_TheiaBXVxquD/resources/app/node_modules/electron-updater/out/main.js
- /tmp/.mount_TheiaBXVxquD/resources/app/node_modules/theia-blueprint-updater/lib/electron-main/update/theia-updater-impl.js
- /tmp/.mount_TheiaBXVxquD/resources/app/node_modules/theia-blueprint-updater/lib/electron-main/update/theia-updater-main-module.js
- /tmp/.mount_TheiaBXVxquD/resources/app/src-gen/frontend/electron-main.js
- /tmp/.mount_TheiaBXVxquD/resources/app/scripts/theia-electron-main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
    at Function.o._resolveFilename (electron/js2c/browser_init.js:281:681)
    at Module._load (internal/modules/cjs/loader.js:690:27)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/tmp/.mount_TheiaBXVxquD/resources/app/node_modules/electron-updater/out/AppUpdater.js:8:20)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)
    at Module.load (internal/modules/cjs/loader.js:815:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/tmp/.mount_TheiaBXVxquD/resources/app/node_modules/electron-updater/out/AppUpdater.js',
    '/tmp/.mount_TheiaBXVxquD/resources/app/node_modules/electron-updater/out/main.js',
    '/tmp/.mount_TheiaBXVxquD/resources/app/node_modules/theia-blueprint-updater/lib/electron-main/update/theia-updater-impl.js',
    '/tmp/.mount_TheiaBXVxquD/resources/app/node_modules/theia-blueprint-updater/lib/electron-main/update/theia-updater-main-module.js',
    '/tmp/.mount_TheiaBXVxquD/resources/app/src-gen/frontend/electron-main.js',
    '/tmp/.mount_TheiaBXVxquD/resources/app/scripts/theia-electron-main.js',
    undefined
  ]
}
```